### PR TITLE
feat(TPG>=6.8.0)!: add KCP log params

### PR DESCRIPTION
**Pull request by DrFaust92**

### CODE REVIEW

File: modules/beta-autopilot-public-cluster/README.md [hunk 1]
```diff
@@ -62,125 +62,6 @@ Then perform the following commands on the root folder:
 - `terraform destroy` to destroy the built infrastructure
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-## Inputs
```
Comment (apeabody) [inline modules/beta-autopilot-public-cluster/README.md:65]: Thanks @DrFaust92! - Something strange with the docs, can you please build again.

Comment (DrFaust92) [reply modules/beta-autopilot-public-cluster/README.md]: fixed

**Comment by apeabody** [comment]
Hi @DrFaust92 - Can you please rebase?  Thanks!

**Comment by DrFaust92** [comment]
[apeabody](https://github.com/apeabody) sorry for delay, done

**Comment by apeabody** [comment]
> [apeabody](https://github.com/apeabody) sorry for delay, done

No worries, thanks!

**Review by apeabody — APPROVED** [review]
Thanks for the contribution @DrFaust92!

**Pull request merged**